src/Core/AzureStorageEmulatorAutomation.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.IO;
+
+namespace RimDev.Automation.StorageEmulator
+{
+    public class AzureStorageEmulatorAutomation : IDisposable
+    {
+        public bool StartedByAutomation { get; private set; }
+
+        public void Dispose()
+        {
+            if (StartedByAutomation)
+            {
+                Stop();
+            }
+        }
+
+        public void Start()
+        {
+            if (!IsEmulatorRunning())
+            {
+                RunWithParameter("start");
+                StartedByAutomation = true;
+            }
+        }
+
+        public void Stop()
+        {
+            RunWithParameter("stop");
+        }
+
+        public void ClearTables()
+        {
+            RunWithParameter("clear table");
+        }
+
+        public static string GetPathToStorageEmulatorExecutable()
+        {
+            var paths = new[]
+            {
+                Path.Combine(AzureSdkDirectory, @"Storage Emulator\AzureStorageEmulator.exe"),
```

Comment:
I'm assuming installations are going to default places. Is there a way to change it as part of the installation (or, at least a *normal* installation, anyhow, if there's special ways to specify the path to install to)?